### PR TITLE
Development

### PR DESCRIPTION
**Pull request by PhMueller**
Upgrade the master to the version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+# HPOBench
 
-HPOlib2 is a library for hyperparameter optimization and black-box optimization benchmark with a focus on reproducibility.
+HPOBench is a library for hyperparameter optimization and black-box optimization benchmark with a focus on reproducibility.
 
-**Note:** Hpolib2 is under active construction. Stay tuned for more benchmarks. Information on how to contribute a new benchmark will follow shortly.
+**Note:** HPOBench is under active construction. Stay tuned for more benchmarks. Information on how to contribute a new benchmark will follow shortly.
 
-**Note:** If you are looking for a different version of HPOlib2, you might be looking for [HPOlib1.5](https://github.com/automl/HPOlib1.5) 
+**Note:** If you are looking for a different version of HPOBench, you might be looking for [HPOlib1.5](https://github.com/automl/HPOlib1.5)
```
Comment (KEggensperger) [inline README.md]: Replace with "If you are looking for a different or older version of our benchmarking library, you might be looking for ..."

File: README.md [hunk 2]
```diff
@@ -78,29 +78,29 @@ pip install .
...
 ### Remove all caches
 
-#### HPOlib data
-HPOlib stores downloaded containers and datasets at the following locations:
+#### HPOBench data
+HPOBench stores downloaded containers and datasets at the following locations:
 
 ```bash
-$XDG_CONFIG_HOME # ~/.config/hpolib2
-$XDG_CACHE_HOME # ~/.cache/hpolib2
```
Comment (KEggensperger) [inline README.md:102]: Sorry, I replaced this directly on master. There should be 3 different directories.

File: hpobench/benchmarks/nas/nasbench_201.py [hunk 3]
```diff
@@ -281,10 +289,14 @@ def objective_function_test(self, configuration: Union[CS.Configuration, Dict],
 
         # The result dict should contain already all necessary information -> Just swap the function value from valid
         # to test and the corresponding time cost
-        result = self.objective_function(configuration=configuration, fidelity=fidelity, data_seed=(777, 888, 999),
+        assert fidelity['epoch'] == 199, 'Only test data for the 200. epoch is available. ' \
```
Comment (KEggensperger) [inline hpobench/benchmarks/nas/nasbench_201.py]: Shouldn't this be 200?

File: hpobench/benchmarks/nas/tabular_benchmarks.py [hunk 4]
```diff
@@ -37,10 +37,10 @@
 import numpy as np
 from tabular_benchmarks.fcnet_benchmark import FCNetBenchmark
 
-import hpolib.util.rng_helper as rng_helper
-from hpolib.abstract_benchmark import AbstractBenchmark
+import hpobench.util.rng_helper as rng_helper
+from hpobench.abstract_benchmark import AbstractBenchmark
 
-__version__ = '0.0.1'
+__version__ = '0.0.3'
```
Comment (KEggensperger) [inline hpobench/benchmarks/nas/tabular_benchmarks.py]: Shouldn't this rather be 0.0.2?

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,6 @@
+# 0.0.5
+  * Rename package to HPOBench
+
```
Comment (KEggensperger) [inline changelog.md]: There is probably a bit more:
  * Added BNN benchmark
  * Updated returned loss values for NB201

File: hpobench/__version__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+__version__ = '0.0.5dev'
```
Comment (KEggensperger) [inline hpobench/__version__.py]: This should be only "0.0.5"

**Review by KEggensperger — CHANGES_REQUESTED** [review]
Thank you. There are a few minor things, but overall this looks good and can be merged.

File: hpobench/__version__.py (hunk 6 above)
Comment (PhMueller) [reply hpobench/__version__.py]: Thought about this too. But if I change it here, then the dev branch loses its _dev tag. So I thought, that we have to replace it directly on the master later, or is there another simple way I don't see? :-)

Comment (KEggensperger) [reply hpobench/__version__.py]: No, we do it here and then directly push a new commit to dev initializing the new version (see here: https://github.com/automl/HPOBench/commit/6ffe1a44d233d57aadee2624acc60cc27dc1c3ec)

**Review by KEggensperger — APPROVED** [review]

**Pull request merged**